lectures/week_07/README.md

```diff
@@ -0,0 +1,108 @@
+
+1. Ensure that Docker Desktop is running. 
+
+2. This command will pull the image `rnabio` to your local Docker client with the tag `0.0.1` from the `griffithlab` DockerHub repository:
+
+```bash
+docker pull griffithlab/rnabio:0.0.1
+```
+
+3. Setup a local workspace directory for the RNAseq course. If you change the path or command used in this Step, please update the path to the workspace directory accordingly in Step 4. Also, make a file `test_my_docker_mount` that we will look for later.
+
+```bash
+mkdir -p bfx-workshop/rnabio-workspace
+echo 'this file helps me test my docker mount' >> bfx-workshop/rnabio-workspace/test_my_docker_mount.txt
+```
+
+4. Enter the directory where you created the `rnabio-workspace` folder, and initialize a Docker container using the image we pulled above. `-v` tells Docker to mount our workspace directory within the Docker container as `/workspace` with read-write priveleges. You'll see in the RNAseq course `/workspace` is the base directory for nearly all commands and steps.
+
+```bash
+cd bfx-workshop/rnabio-workspace
+docker run -v $PWD/:/workspace:rw -it griffithlab/rnabio:0.0.1 /bin/bash
+```
+
+5. Use `ls` to see what's in your current directory, then enter the `workspace` folder and use `ls` again to see what is in the `workspace` folder.
+
+```bash
+ls
+cd workspace
+ls
+```
+
+### User Setup
+
+Now that we are running a Docker container, Docker, by default, will log you in as the "root" user. We need to run as the ubuntu user to match the RNAseq course tutorials.
+
+1. Switch User `su` to the ubuntu user:
+
+```bash
+su ubuntu
+```
+
+2. Source the pre-installed `.bashrc` file to configure your environment to match the RNAseq course:
+
+```bash
+source ~/.bashrc
+```
+
+NOTE: Using Docker and the persistent "workspace" volume we attached will allow you to start/stop as you wish. EVERYTIME YOU LOGIN TO THE DOCKER CONTAINER, YOU MUST LOGIN AS THE `ubuntu` USER *AND* `source ~/.bashrc` UPON EACH LOGIN.
+
+### Environment Setup
+
+Create a working directory and set the ‘RNA_HOME’ environment variable
+```
+mkdir -p ~/workspace/rnaseq/
+
+export RNA_HOME=~/workspace/rnaseq
+```
+
+Make sure whatever the working dir is, that it is set and is valid
+```
+echo $RNA_HOME
+```
+
+Since all the environment variables we set up for the RNA-seq workshop start with ‘RNA’ we can easily view them all by combined use of the env and grep commands as shown below. The env command shows all environment variables currently defined and the grep command identifies string matches.
+```
+env | grep RNA
+```
+
+In order to view the contents of this file, you can type:
+```
+less ~/.bashrc
+```
+To exit `less`, type `q`.
+
+### Known Issues/ Discrepancies from RNAbio website
```

Comment:
At some point, we should consider forking the RNAbio content and just incorporating these few changes. It could be a "local version" of RNAbio that would be useful for lots of folks that can't spin up an AMI. If you don't have time this year, that's fine, but something to think about for the future.

Reply:
I didn't get around to seeing these comments until now, but happy to work on this! 

For the local version part, would that be similar to how we use the `griffithlab/rnabiodev:0.0.3` docker when we're testing updates to the website?